google/resource_container_cluster.go

```diff
@@ -149,6 +149,21 @@ func resourceContainerCluster() *schema.Resource {
 				Elem:     &schema.Schema{Type: schema.TypeString},
 			},
 
+			"legacy_abac": {
```

Comment:
Yeah, I was leaving the option open in case any others get added, but realistically I don't think they will so I went ahead and made the change.